.storybook/components/preview/Preview.css

```
@@ -1,8 +1,8 @@
 /* PREVIEW */
 .o-ui-sb-preview {
     margin: var(--o-ui-sp-7) 0 var(--o-ui-sp-8);
-    border-radius: var(--o-ui-br-4);
-    box-shadow: var(--o-ui-bs-alias-raised);
+    border-radius: var(--hop-shape-rouded-md);
```

```suggestion
    border-radius: var(--hop-shape-rounded-md);
```
There is a spelling mistake